docs/source/markdown/podman-run.1.md

```diff
@@ -92,8 +92,9 @@ If the host uses cgroups v1, the default is set to **host**.  On cgroups v2 the
 **--cgroups**=*mode*
 
 Determines whether the container will create CGroups.
-Valid values are *enabled* and *disabled*, which the default being *enabled*.
+Valid values are *enabled*, *disabled*, *disabled-conmon*, which the default being *enabled*.
```

Comment:
I think `no-conmon` would be a better name, but I'd like other folks to agree to that before we change it.

Reply:
@vrothberg do you have any preference on the naming?

Reply:
I don't have a preference.